specification/e-referrals-service-api.yaml

```diff
@@ -200,6 +200,8 @@ info:
   license:
     name: MIT
 servers:
+  - url: 'https://internal-dev-sandbox.api.service.nhs.uk/referrals-pr-117/FHIR'
```

Comment:
This should be removed before these changes can be merged into the shared branch.